content/docs/components-and-props.md

```diff
@@ -42,25 +43,28 @@ class Welcome extends React.Component {
 }
 ```
 
-The above two components are equivalent from React's point of view.
+माथिका दुई componentहरू Reactको  दृष्टिकोणमा बराबर छन्।
+
+
+Classहरूका केहि अतिरिक्त विशेषताहरु  पनि छन् जुन हामी [अर्को खण्डमा](/docs/state-and-lifecycle.html) छलफल गर्नेछौं। त्यतिन्जेल, हामी तिनीहरूको संक्षिप्तताका लागि function components प्रयोग गर्नेछौं।
 
-Classes have some additional features that we will discuss in the [next sections](/docs/state-and-lifecycle.html). Until then, we will use function components for their conciseness.
 
 ## Rendering a Component {#rendering-a-component}
 
-Previously, we only encountered React elements that represent DOM tags:
+अहिले सम्म,हामीलाई React elementsले  DOM ट्यागको प्रतिनिधित्व गरेको बारेमा मात्र  थाहा छ:
+:
 
 ```js
 const element = <div />;
 ```
 
-However, elements can also represent user-defined components:
+यद्यपि React elementsले user-defined componentहरूको पनि प्रतिनिधित्व गर्न सक्दछन्:
```

Comment:
I don't have a huge opinion on this, but would it make sense to translate user-defined? Maybe something along the lines of `प्रयोगकर्ताद्वारा  परिभाषित component...` ?